CRM/Contact/Page/View/Relationship.php

```diff
@@ -186,6 +190,17 @@ public function preProcess() {
 
     $this->_action = CRM_Utils_Request::retrieve('action', 'String', $this, FALSE, 'browse');
     $this->assign('action', $this->_action);
+
+    // @todo implement something like entityFormTrait (entityPageTrait?)
+    $this->assign('entityInClassFormat', strtolower(str_replace('_', '-', $this->getDefaultEntity())));
```

Comment:
I think it's worth adding the entityPageTrait like you say . - even if it just has this one fn initially

Reply:
done